ios/ReactNativeCameraKit/SimulatorCamera.swift

```diff
@@ -164,8 +164,8 @@ class SimulatorCamera: CameraProtocol {
     
 
     func isBarcodeScannerEnabled(_ isEnabled: Bool,
-                                 supportedBarcodeType: [AVMetadataObject.ObjectType],
-                                 onBarcodeRead: ((_ barcode: String) -> Void)?) {}
+                                 supportedBarcodeTypes supportedBarcodeType: [AVMetadataObject.ObjectType],
```

Comment:
Let's remove old `supportedBarcodeType`

Shouldn't we update this type as well, so it takes `CodeFormat` as input and output?

```suggestion
                                 supportedBarcodeTypes: [CodeFormat],
```